client/src/components/CBtn.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Button from './CBtn.style';
+
+const CBtn = ({ text, path }) => (
+  <Button onClick={path} type="button">
```

Comment:
![Screen Shot 2019-06-21 at 7 24 29 PM](https://user-images.githubusercontent.com/9094166/59943521-5d03ac00-945a-11e9-90a6-71727064d5e9.png)
Just tried it out and I'm getting an error that the onClick should be a function, but is instead a string. That's because we're just passing in the path by itself. This is easily fixed though!

We just need to create a small function as part of the component that is called onClick, with the path as an argument.

In the old login component, the button used this function: 
```
login = () => {
    const err = this.validate();
    const { email, password } = this.state;
    if (!err) {
      const { history } = this.props;
      const inputs = {
        email,
        password,
      };
```

We're going to need to implement something similar (maybe without the validation) in the button component!

Reply:
Can you help me with this?
I created a function and called it `OnClick()` I want to ask what put in the function.
Can you see what I did??

@Oliversw

Reply:
Just had a look and realized that it is more complicated than I had realized! We have some buttons that only need to take the user to another page, but also buttons like login and signup that require running functions like validation and api calls before changing the page. Since we already have those functions written, I just changed the button component back to a functional component, and defined button text and a callback as props. I think that will be the easiest way to do it, we just have to make sure to always write a function that we're passing to the button so that it works properly. Does that make sense?

Reply:
Yes, it is clear @Oliversw 
I want to make merge for `PR`

Reply:
Merge whenever you like @shaima96, it's approved! 😄